### PR TITLE
modify CIT api2.0 sel tests to align with the new event format

### PR DESCRIPTION
**Pull request by iceiilin**
modify AMQP message format in api 2.0 sel alert tests to align with the new event format design.
@sunnyqianzhang @nortonluo @anhou @pengz1 

jenkins: depend on https://github.com/RackHD/on-tasks/pull/380

### CODE REVIEW

**Comment by iceiilin** [comment]
test this please

**Comment by pengz1** [comment]
+1

**Comment by iceiilin** [comment]
test this please

File: test/tests/api/v2_0/sel_alert_poller_tests.py [hunk 1]
```diff
@@ -188,10 +191,12 @@ def test_full_sel(self):
...
             self.run_ipmitool_command((n['node_ip']), "sel add /tmp/selError.txt")
             #time.sleep(1)
+            self.__event_count = 0
+            self.__event_total = n["available_sel_entries"]
+
             self.__task_worker.start()
 
-            assert_equal(len(self.__amqp_alert["value"]["alerts"]), n["available_sel_entries"])
-            #self.__amqp_alert = {}
+            assert_equal(self.__event_count, n["available_sel_entries"])
```
Comment (nortonluo) [inline test/tests/api/v2_0/sel_alert_poller_tests.py]: If SEL entry is added N entrys by ipmitool cmd, the poller is polling in 1 minute interval. Will this lead to N times AMQP alert?

Comment (iceiilin) [inline test/tests/api/v2_0/sel_alert_poller_tests.py]: N SEL events will be sent one time by  the poller

Comment (iceiilin) [inline test/tests/api/v2_0/sel_alert_poller_tests.py]: Right. N AMQP alerts will be sent in one poller time

**Comment by iceiilin** [comment]
test this please

**Comment by changev** [comment]
test this please

**Comment by nortonluo** [comment]
+1

**Comment by iceiilin** [comment]
test this please

**Comment by iceiilin** [comment]
test this please

**Comment by iceiilin** [comment]
test this please

**Comment by iceiilin** [comment]
test this please

**Pull request merged**